app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -4,57 +4,121 @@
   <data>
 
+    <import type="android.view.View" />
+
+    <import type="org.oppia.app.model.ChapterPlayState" />
+
     <variable
       name="viewModel"
       type="org.oppia.app.story.storyitemviewmodel.StoryChapterSummaryViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="350dp"
-    android:layout_height="wrap_content"
-    android:layout_marginStart="10dp"
-    android:layout_marginTop="10dp"
-    android:padding="16dp"
-    android:onClick="@{viewModel.onExplorationClicked}"
-    android:contentDescription="@{viewModel.name}"
-    app:roundedRectDrawableWithColor="@{viewModel.chapterThumbnail.backgroundColorRgb}">
-
-    <ImageView
-      android:id="@+id/chapter_thumbnail"
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <com.google.android.material.card.MaterialCardView
       android:layout_width="match_parent"
-      android:layout_height="180dp"
-      android:paddingStart="40dp"
-      android:paddingTop="40dp"
-      android:paddingEnd="40dp"
-      android:scaleType="centerCrop"
-      android:src="@{viewModel.chapterThumbnail.thumbnailGraphic}"
-      app:layout_constraintBottom_toTopOf="@id/chapter_title"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toTopOf="parent" />
-
-    <TextView
-      android:id="@+id/chapter_title"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_margin="8dp"
-      android:fontFamily="sans-serif-medium"
-      android:text="@{viewModel.name}"
-      android:textColor="@color/black"
-      android:textSize="14sp"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/chapter_thumbnail" />
-
-    <TextView
-      android:id="@+id/chapter_summary"
-      android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_margin="8dp"
-      android:fontFamily="sans-serif-medium"
-      android:text="@{viewModel.summary}"
-      android:textColor="@color/black"
-      android:textSize="12sp"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/chapter_title" />
-  </androidx.constraintlayout.widget.ConstraintLayout>
+      android:layout_margin="16dp"
+      android:clickable="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES}"
+      android:onClick="@{viewModel.onExplorationClicked}"
+      app:cardCornerRadius="4dp"
+      app:cardElevation="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @dimen/elevation_4dp : @dimen/elevation_0dp}"
+      app:strokeColor="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/chapterCardGreenBorder : @color/chapterCardGreyBorder}"
```

Comment:
Color doesn’t seems matching with the mock.

Reply:
This again relates to the comment earlier, the mocks that you are checking are v7_draft and and the mocks on which this UI has been made is v6_final and because of that there is confusion. Waiting for @mschanteltc for correct v6_spec link

Reply:
This has been resolved again. The mocks were updated when you were checking the PR, now they are back to original. So this does not need any change.